main.go

```diff
@@ -428,15 +454,18 @@ func installOption(listAll bool, custBinPath, mirrorURL *string) {
 
 // install when tf file is provided
 func installTFProvidedModule(dir string, custBinPath, mirrorURL *string) {
-	fmt.Printf("Reading required version from terraform file\n")
+	fmt.Println("Reading required version from terraform file")
 	module, _ := tfconfig.LoadModule(dir)
-	tfconstraint := module.RequiredCore[0] //we skip duplicated definitions and use only first one
+	if len(module.RequiredCore) >= 1 {
```

Comment:
Wrong conditional RHS copied from another function?
```suggestion
	if len(module.RequiredCore) == 0 {
```

Reply:
Yes, should be "== 0" for sure.